wagtail_localize/static_src/common/components/SnippetChooser/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FunctionComponent } from 'react';
+import gettext from 'gettext';
+
+interface SnippetAPI {
+    id: string | number;
+    title: string;
+    edit_url: string;
+}
+
+interface SnippetChooserProps {
+    snippetModel: {
+        app_label: string;
+        model_name: string;
+        verbose_name: string;
+        verbose_name_plural: string;
+    };
+    snippetId: string | number | null;
+}
+
+const SnippetChooser: FunctionComponent<SnippetChooserProps> = ({
+    snippetModel,
+    snippetId
+}) => {
+    const [snippetInfo, setSnippetInfo] = React.useState<SnippetAPI | null>(
+        null
+    );
+
+    React.useEffect(() => {
+        setSnippetInfo(null);
+
+        if (snippetId) {
+            fetch(
+                `/admin/localize/api/snippets/${snippetModel.app_label}/${snippetModel.model_name}/${snippetId}/`
```

Comment:
An idea since /admin/ is hard coded but places like mozilla use /cms/ instead (if this is even a problem)...

In a package I made last week I set `window.adminUrl = '{% url 'wagtailadmin_home' %}'` and prepended these URLS in the JavaScript with `window.adminUrl`. 

^ If this is not a problem, totes feel free to ignore :)

Reply:
Thanks! I'll solve this in the next PR as there are a few other places to fix too